lib/ex_unit_properties.ex

```diff
@@ -637,7 +651,7 @@ defmodule ExUnitProperties do
     Enum.map_join(values, "\n\n", fn {gen_string, value} ->
       String.trim_trailing("""
       * Clause:    #{gen_string}
-        Generated: #{inspect(value)}
+        Generated: #{inspect(value, Application.get_env(:stream_data, :inspect_opts, []))}
```

Comment:
Let's add the default (`[]`) to `application/0` in `mix.exs` and call `Application.fetch_env!/2` here instead.

Reply:
Done. I learned something new today ;-)